bats_test/base-reward/base-reward.bats

```diff
@@ -9,3 +9,8 @@ load '/home/node/packages/node_modules/bats-assert/load.bash'
     run -0 yarn ardrive base-reward
     assert_output --regexp '^[0-9]+$'
 }
+
+@test "'ardrive base-reward --gateway' will error with an invalid Arweave gateway" {
+    run yarn ardrive base-reward --gateway http://fakeway.net
```

Comment:
Nit: If `fakeway.net` ever becomes real, this test could break...

Reply:
Same comment for all the other usages of fakeway.net in this PR.

Reply:
Changed this to `http://localhost:1337`: [033556a](https://github.com/ardriveapp/ardrive-cli/pull/246/commits/033556a9c53f986d9e182af94521e6a908c4aa1e)